### PR TITLE
fix: scroll to bottom when sending a message

### PR DESCRIPTION
**Pull request by ohassine**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Sending a message while reading older messages does not scroll the view to the message that just I sent

### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

Scroll to bottom when sending messages.

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1259/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5282310)(5282310)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5282310)(5282310)-dev-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt [hunk 1]
```diff
@@ -461,8 +460,18 @@ private fun ConversationScreenContent(
                 onShowContextMenu = onShowContextMenu
             )
         },
-        onSendTextMessage = onSendMessage,
-        onSendAttachment = onSendAttachment,
+        onSendTextMessage = { message, mentions, messageId ->
+            scope.launch {
+                lazyListState.animateScrollToItem(0)
```
Comment (vitorhugods) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt]: What if user is seeing some really old message? Will this animation take ages?

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt [hunk 2]
```diff
@@ -461,8 +460,18 @@ private fun ConversationScreenContent(
...
+            }
+            onSendMessage(message, mentions, messageId)
+        },
+        onSendAttachment = {
+            scope.launch {
+                lazyListState.animateScrollToItem(0)
```
Comment (vitorhugods) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt]: Same here

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt (hunk 1 above)
Comment (ohassine) [reply app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt]: Good point... I will update it to `scrollToItem` which Instantly brings the item to 0

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1259/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5283950)(5283950)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5283950)(5283950)-dev-compat.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1259/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5284128)(5284128)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5284128)(5284128)-dev-compat.apk)

**Review by MohammedMokresh — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1259/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5343648)(5343648)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5343648)(5343648)-dev-compat.apk)

**Pull request merged**